src/app/pages/AccountDetailsPage/index.tsx

```diff
@@ -30,3 +34,11 @@ export const AccountDetailsPage: FC = () => {
     </PageLayout>
   )
 }
+
+export const AccountDetailsView: FC<{
+  isLoading: boolean
```

Comment:
We have this type defined. Do we need to duplicate it?

Reply:
I would keep it decoupled, unless we rename Account component to AccountDetailsView (and later split out Account component if we ever need a more flexible reusable one)